Python/Module1_GettingStartedWithPython/GettingStartedWithPython.md

```diff
@@ -144,6 +144,39 @@ As such, Python is a language that is conducive to rapidly prototyping and testi
 We will be relying heavily on Python and a library for doing optimized numerical computations, called NumPy, throughout this course.
 <!-- #endregion -->
 
+<!-- #region -->
+## Keeping Up To Date
+
+New versions of Python come out periodically, bringing new features and fixes. 
+The most recent version of Python is Python 3.8.4.
```

Comment:
I'd generally advise against these kinds of statements because it's going to be either (1) a pain to update every time a new release comes out or (2) just be outdated